Dockerfile

```
@@ -1,33 +1,140 @@
+ARG clamav_version=1.3.1
 ARG ruby_version=3.3
 ARG base_image=ghcr.io/alphagov/govuk-ruby-base:$ruby_version
 ARG builder_image=ghcr.io/alphagov/govuk-ruby-builder:$ruby_version
-ARG clam_engine=clamav-1.2.1.linux.x86_64.deb
 
-FROM --platform=$TARGETPLATFORM $builder_image AS builder
+FROM --platform=$TARGETPLATFORM index.docker.io/library/ubuntu:22.04 AS clam_builder
```

```suggestion
FROM --platform=$TARGETPLATFORM public.ecr.aws/lts/ubuntu:22.04_stable AS clam_builder
```

Maybe keep the base the same as our ruby images?
https://github.com/alphagov/govuk-ruby-images/blob/e0e9bd1aaf52ba4d5d5c7f82c10662922d9a1d41/base.Dockerfile#L1